pyproject.toml

```
@@ -77,7 +77,12 @@ checksumdir = "^1.2.0"
 importlib-metadata = "^4.13"
 
 [tool.poetry.extras]
-pydantic = ["pydantic", "email-validator"]
+parser = ["pydantic", "email-validator", "jmespath"]
+validation = ["fastjsonschema", "jmespath"]
```

```suggestion
parser = ["pydantic", "email-validator"]
validation = ["fastjsonschema"]
```

I was wrong. We're not gonna need that anymore because it comes with botocore. If boto ever removes we'll catch it in our E2E tests now.

This wasn't working either because we didn't specify the dependency `jmespath` like we did for `pydantic` and others.